### DISCUSSION TITLE
Theming the tray icon (AppImage version)

### DISCUSSION THREAD

**Discussion by Bingo90** [bpozdena/OneDriveGUI#175] [Q&A]
Is it possible to theme the tray icon with an icon theme when using the AppImage version? I'm honestly not too familiar with AppImages, this is the only program I'm using that's using that format ^^

**Comment by bpozdena** [comment]
Unfortunately not natively. You can however replace the icons under `src/resources/images/` if you'd like.  Note that the upcoming version 1.3.0 will dynamically change system tray icon based on the client state/status.
